tls_codec/src/quic_vec.rs

```diff
@@ -214,7 +215,7 @@ fn write_hex(f: &mut fmt::Formatter<'_>, data: &[u8]) -> fmt::Result {
 /// Use this struct if bytes are encoded.
 /// This is faster than the generic version.
 #[cfg_attr(feature = "serde", derive(SerdeSerialize, SerdeDeserialize))]
-#[derive(Clone, PartialEq, Eq, Hash, Ord, PartialOrd)]
+#[derive(Clone, PartialEq, Eq, Hash, Ord, PartialOrd, Zeroize)]
```

Comment:
So only zeroize doesn't make a performance difference?

Reply:
Benching locally I don't see a meaningful difference. The way I understand it, deriving `Zeroize` only gives you the ability to zeroize manually. Automatically zeroizing on drop seems to cause performance issues.